src/c/user-contributed.js

```diff
@@ -0,0 +1,88 @@
+import m from 'mithril';
+import models from '../models';
+import postgrest from 'mithril-postgrest';
+import _ from 'underscore';
+import h from '../h';
+import userVM from '../vms/user-vm';
+import projectCard from './project-card';
+
+const userContributed = {
+    controller(args) {
+        const contributedProjects = m.prop(),
+              user_id = args.userId,
+              pages = postgrest.paginationVM(models.project),
+              loader = m.prop(true),
+              contextVM = postgrest.filtersVM({
+                  project_id: 'in'
+              });
+
+        userVM.getUserContributedProjects(user_id, null).then((data) => {
+            contributedProjects(data);
+            contextVM.project_id(_.map(contributedProjects(), (project) => {return project.project_id;})).order({
```

Comment:
@vicnicius i think that is because we use a array of project_ids from contributors before get on projects.